packages/chart/src/ChartUtils.ts

```diff
@@ -1877,12 +1855,67 @@ class ChartUtils {
     return axis;
   }
 
+  /**
+   * Creates a plotly layout template object based on a given theme.
+   * See https://plotly.com/javascript/reference/layout/#layout-template
+   * @param theme The theme to use for the layout template
+   */
+  makeDefaultTemplate(theme: Partial<ChartTheme>): Template {
+    /* eslint-disable camelcase */
+    const {
+      error_band_line_color,
+      ohlc_increasing,
+      ohlc_decreasing,
+      title_color,
+    } = theme;
+
+    return {
+      data: {
+        bar: [
+          {
+            marker: {
+              line: { color: 'transparent' },
+            },
```

Comment:
This is the only part I didn't see in the old series styling. Is it intentional?

Reply:
Yes, this is part of removing borders from histograms and gets rid of the default white border.